src/chrome/content/rules/eyeonx.ch.xml

```diff
@@ -0,0 +1,30 @@
+<!--
+		Cert expired:
+		-	event.eyeonx.ch
```

Comment:
every subdomain of eyeonx.ch is answered for by a wildcard dns on deliberate typos.
What about exactly this domain?